packages/terra-action-header/src/clinical-lowlight-theme/ActionHeader.module.scss

```diff
@@ -0,0 +1,29 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .clinical-lowlight-theme {
+    // .header-icon &.back
```

Comment:
Personal opinion, I think mapping the classes is a little over overkill and doesn't provide a ton of value. It's easier to search for the theme variable than the class selector. Generic comments might be a little more robust. I don't think this is harmful if the engineer doing the uplift wants to add them, but I don't think this should be something we enforce when we start uplifting the other components. 

"Back Icon and Hover Back Icon"

Reply:
yeah it's probably personal preference for sure. What exists today with the theme variables is impossible for me to wrap my head around so I listed them here in the class+abc order the appear in the original style sheet so I didn't miss one 😃 

I can remove them if you want